src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -122,24 +123,27 @@ const char* GetCalendarName(CalendarId calendarId)
 		case JAPANESELUNISOLAR:
 		case TAIWANLUNISOLAR:
 			return "chinese";
```

Comment:
So should we not support any of these calendars in these 2 mapping functions?  Note that in CLDR, the "chinese" calendar is associated with some regions in the calendarPreferenceData:
CN, CX, HK, MO, SG, and TW.

Also note that the GetCalendarName(CalendarId) is used to get things like weekday and month names, and getting the native calendar name.  If JapaneseLunisolar gets passed in to GetMonthNames, which CLDR calendar should we be using?